server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -209,7 +209,9 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
             operationsDbMongoManager,
             config.get("mongo:collectionNames:reservations"));
 
-        const tenantManager = new services.TenantManager(authEndpoint);
+        const defaultHistorianUrl = config.get("worker:blobStorageUrl");
+        const defaultInternalHistorianUrl = config.get("worker:internalBlobStorageUrl") || defaultHistorianUrl;
```

Comment:
I make the name consistent as the `defaultInternalHistorianUrl` and `defaultHistorianUrl` since they are actually the default names of the service. We will always firstly fetch these default URLs.

Reply:
I removed `defaultHistorianUrl` since they are not needed in our case. Meanwhile, I make all of the `defaultInternalHistorianUrl` as `internalHistorianUrlOverride` to be consistent.